### ISSUE TITLE
getting errors 

### ISSUE THREAD

**Issue by WolfoTheWolf** [Angush/trellobot#16]
Im just getting these errors any help?

![image](https://user-images.githubusercontent.com/56661068/102307431-747a0300-3f32-11eb-8752-36f326a79135.png)

**Comment by DavideTW** [comment]
Solved with commit cc380a8.

**Comment by Benbentwo** [comment]
Agreed, solved with #17. apply this change to local / server and rerun process
